src/microscope_laser_autofocus/README.md

```diff
@@ -0,0 +1,40 @@
+# Microscope Laser Autofocus with the WDI ATF sensor and C#
+
+*By Stefan Martin*
+
+## Dependencies
+
+FIXME: Talk about the hardware setup here.
```

Comment:
So on my first pass I assumed this was going to be a regular example and added this readme. If not maybe you want to present the instructions to them differently and remove this file.

Reply:
I am going to leave the information about where to get ATF_LIB external to the example code since this won't be a publicly available example.

Reply:
I've added description of hardware setup.